Realm/Realm/Handles/SessionHandle.cs

```diff
@@ -28,7 +28,7 @@
 using Realms.Sync.ErrorHandling;
 using Realms.Sync.Exceptions;
 using Realms.Sync.Native;
-using CompensatingWriteInfo = Realms.Exceptions.CompensatingWriteInfo;
+using CompensatingWriteInfo = Realms.Sync.Exceptions.CompensatingWriteInfo;
```

Comment:
```suggestion
```
You don't need this anymore as `Realms.Sync.Exceptions` is already used above.

Reply:
We do because there's a native `CompensatingWriteInfo` struct as well as the public-facing object of the same name we return to users. Without this line, I'd need to fully qualify usages of `CompensatingWriteInfo`.

Reply:
Derp. I missed that. Makes sense.

Reply:
Moved the native struct to a nested struct and now we really don't need it.